openldap/Makefile

```diff
@@ -0,0 +1,66 @@
+##
+## (C) Copyright 2017 Nuxeo (http://nuxeo.com/) and others.
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##
+## Contributors:
+##     Frantz FISCHER <ffischer@nuxeo.com>
+
+SHELL=/bin/bash
+TAG-NAME=nuxeo/openldap
+NAME=dockerpriv.nuxeo.com:443/$(TAG-NAME)
+VERSION=1.0.0
+SCM-REF=PIPELINE-SCM-VALUE
+
+all: clean build test
+
+clean:
+	-docker-compose --file tests/docker-compose.yml rm -f
+	-docker rm $(docker ps --filter=status=exited --filter=status=created -q) 2>/dev/null
```

Comment:
Please add an echo after the first line to warn about the unexpected following cleanup